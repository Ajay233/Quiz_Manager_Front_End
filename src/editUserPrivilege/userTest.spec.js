import React from 'react'
import User from './user'
import renderer from 'react-test-renderer'
import configureStore from 'redux-mock-store'
import thunk from 'redux-thunk'
import history from '../history'

describe("User", () => {
  it("should render a user's details using the props passed to it", () => {
    const userResults = {
      forename: "testForename",
      surname: "testSurname",
      email: "testEmail",
      permission: "USER",
      verified: "true"
    }
    const component = renderer.create(
      <User userResults={userResults} />
    )

    expect(component).toMatchSnapshot()
  })
})
