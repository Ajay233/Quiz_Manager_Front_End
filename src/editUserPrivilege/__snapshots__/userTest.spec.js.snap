// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User should render a user's details using the props passed to it 1`] = `
<div>
  <div>
    Name: testForename testSurname
  </div>
  <div>
    Email: testEmail
  </div>
  <div>
    Permission: USER
  </div>
  <div>
    Verified: No
  </div>
</div>
`;
